WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -49,7 +50,8 @@ class PostListItemViewHolder(
         setTextOrHide(tvTitle, item.data.title)
         setTextOrHide(tvExcerpt, item.data.excerpt)
         setTextOrHide(tvDateAndAuthor, item.data.dateAndAuthor)
-        setTextOrHide(tvStatusLabels, item.data.statusLabels)
+        uiHelpers.updateVisibility(tvStatusLabels, item.data.statusLabels.isNotEmpty())
```

Comment:
I was under the impression that we have abandoned Hungarian notation. Is the `tv` prefix _new_ or we're just keeping old property names? Why wouldn't `"Labels"` be enough?

Reply:
AFAIK we decided not to use Hungarian notation for encoding visibility modifiers. However, I'll be happy to remove even this usage of the Hungarian notation if you think the code will be cleaner. I felt like it's easier to follow we are working with a view and not with data.

Reply:
The only decision made re: prefix notation has been to continue using `m` and `s` prefixes for member and static fields in Java code, but not use them for Kotlin code. Everything else is up to preference/readability.

Reply:
No need to change it. As long as we're all consistent. My past experience was the same anyway:

```kt
// instead of
val tvTitle: TextView

// I used 
val titleTextView: TextView
```

It's longer. 😆 But I guess it's easier to grasp for newcomers.

Reply:
I guess you are right, the second convention is more common in our codebase. I'll update it in a separate PR along with `statusLabels` -> `statuses`.